packages/openapi-v3/test/unit/controller-spec/controller-decorators/request-body-decorators/shortcut.test.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {getControllerSpec, requestBody, post} from '../../../../../';
+
+describe('requestBody decorator - shortcuts', () => {
+  context('array', () => {
+    it('', () => {
```

Comment:
Im assuming you left this empty because the test will read `requestBody decorator - shortcuts array`? What if we left `context` out and have `array` in the `it` block since there is only one test?

Reply:
ah good catch.

And the `context` if for more shortcuts in the future, see issue https://github.com/strongloop/loopback-next/issues/1053